src/main/groovy/com/github/jrubygradle/jar/JRubyJarTask.groovy

```diff
@@ -19,4 +19,14 @@ class JRubyJar extends Jar {
     JRubyJar() {
```

Comment:
Your file is name `JRubyJarTask.groovy`, but the class is `JRubyJar`. Although Groovy allows this, it is best to stick with the Java convention.Renaming file to `JRubyJar.groovy` would thus be better.